test/stream.js

```diff
@@ -49,74 +64,82 @@ test('Writatable Stream API', function(t) {
 })
 
 test('Error conditions', function(t) {
-  var feed = new follow.Changes
-  t.throws(write, 'Throw if the feed type is not defined')
+  // CouchDB 1.X only test
```

Comment:
Again why is this a 1.x only test?

Reply:
After a bit of jiggery-pokery this now passes.